amespahdbpythonsuite/tests/test_parsers.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""
+test_parsers.py
```

Comment:
For pyPAHdb we didn't use pytest; I like to keep things consistent between the different code-bases; either change here or in pyPAHdb---pytest is probably the preferred way ...